fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -11,8 +11,13 @@ def self.run(params)
 
         if command_name == "archive" && params[:frameworks].count > 0
           cmd.concat(params[:frameworks])
-        elsif ["update", "build", "bootstrap"].include?(command_name) && params[:dependencies].count > 0
-          cmd.concat(params[:dependencies])
+        elsif ["update", "build", "bootstrap"].include?(command_name)
+          if params[:dependencies].count > 0
+            cmd.concat(params[:dependencies])
```

Comment:
A minor suggestion to make it a bit more Ruby way 😉 

```suggestion
            cmd.concat(params[:dependencies]) if params[:dependencies].count > 0
```

Reply:
Ah nice! Ruby isn't my "primary language".